array-map/main.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-console, no-unused-vars */
+
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const languages = [
+  'Hypertext Markup Language',
+  'Cascading Style Sheets',
+  'ECMAScript',
+  'PHP Hypertext Preprocessor',
+  'Structured Query Language'
+];
+
+const doubled = numbers.map(numbers => numbers * 2);
+console.log('doubled:', doubled);
+
+const prices = numbers.map(numbers => `$${numbers.toFixed(2)}`);
+console.log('prices:', prices);
+
+const upperCased = languages.map(languages => languages.toUpperCase());
+console.log('upperCased:', upperCased);
+
+const firstLetters = languages.map(languages => languages[0]);
```

Comment:
I recommend using the singular form of your array name as your arrow function parameter name, since it's only one element at a time.